fivetran_provider_async/hooks.py

```diff
@@ -222,3 +680,14 @@ async def get_last_sync_async(self, connector_id, xcom=""):
 
 def _retryable_error_async(exception: ClientResponseError) -> bool:
     return exception.status >= 500
+
+
+def _retryable_error(exception) -> bool:
```

Comment:
We should add logic for exponential retry in http requests

Reply:
maybe in other, I don't want to introduce any. feature in this PR, This just copy the code from  https://github.com/fivetran/airflow-provider-fivetran but I can created an issue to track it

Reply:
created https://github.com/astronomer/airflow-provider-fivetran-async/issues/36 I have added the reference to this feedback but feel free to add details. Thank you!